frontend/scss/components/templates/component-detail.scss

```diff
@@ -49,6 +49,19 @@ body {
   .#{utility('toc')} + .#{utility('content')} {
     @include content-with-sidebar;
   }
+
+  .#{utility('versions')} {
```

Comment:
Can you add a bit more margin between the different versions?

Reply:
done